patches/minecraft/net/minecraft/client/renderer/color/ItemColors.java.patch

```diff
@@ -0,0 +1,48 @@
+--- ../src-base/minecraft/net/minecraft/client/renderer/color/ItemColors.java
++++ ../src-work/minecraft/net/minecraft/client/renderer/color/ItemColors.java
+@@ -16,7 +16,6 @@
+ import net.minecraft.nbt.NBTBase;
+ import net.minecraft.nbt.NBTTagIntArray;
+ import net.minecraft.potion.PotionUtils;
+-import net.minecraft.util.ObjectIntIdentityMap;
+ import net.minecraft.util.math.BlockPos;
+ import net.minecraft.world.ColorizerGrass;
+ import net.minecraft.world.IBlockAccess;
+@@ -26,7 +25,9 @@
+ @SideOnly(Side.CLIENT)
+ public class ItemColors
+ {
+-    private final ObjectIntIdentityMap<IItemColor> field_186732_a = new ObjectIntIdentityMap(32);
++    // private final ObjectIntIdentityMap<IItemColor> mapItemColors = new ObjectIntIdentityMap(32); Vanilla
++    // FORGE
```

Comment:
IF you do a comment something like // Forge: Changed from ObjectIntIdeneityMap to HashMap because IDs can change. Or something that is more actually descriptive then that.